models/staging/stg_contract_code.yml

```diff
@@ -0,0 +1,117 @@
+version: 2
+
+models:
+  - name: stg_contract_code
+    tests:
+      - dbt_utils.recency:
+          datepart: day
+          field: cast(closed_at as timestamp)
+          interval: 2
+          meta:
+            description: "Monitors the freshness of your table over time, as the expected time between data updates."
+      - dbt_utils.unique_combination_of_columns:
+          combination_of_columns:
+            - contract_code_hash
+            - contract_code_ext_v
```

Comment:
Just wondering:  is `contract_code_ext_v` needed to make a unique combo? That surprises me. I would have expected just code_hash + ledger + change to be enough

Reply:
Resolved in https://github.com/stellar/stellar-dbt-public/pull/100/commits/e41f9a26bc1044392dc3c92eba23a10f8d995fad